build.sbt

```diff
@@ -151,9 +151,16 @@ lazy val test = crossProject(JSPlatform, JVMPlatform)
   .dependsOn(core, streams)
   .settings(stdSettings("zio-test"))
   .settings(
-    libraryDependencies ++= Seq(
-      "org.portable-scala" %%% "portable-scala-reflect" % "0.1.0"
-    )
+    scalacOptions += "-language:experimental.macros",
+    libraryDependencies ++= {
+      if (isDotty.value)
+        Seq("org.portable-scala" %%% "portable-scala-reflect" % "0.1.0")
```

Comment:
This dependency occurs in both cases, so perhaps you can push the conditional lower, e.g. `++= Seq("org...") ++ if (isDotty.value()) Seq() else Seq(...)`

Reply:
Done.